.github/workflows/build.yml

```diff
@@ -183,15 +184,22 @@ jobs:
           export EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID="${{secrets.EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID}}"
           export EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY="${{secrets.EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY}}"
           if [ ${{ steps.otel-check.outputs.run_opentelemetry }} == "true" ]; then
-            # Set to 'otlp' to enable OpenTelemetry
-            export SYNAPSE_OTEL_INTEGRATION_TEST_EXPORTER="otlp"
+            # Set to 'file' to enable OpenTelemetry export to file
```

Comment:
Not quite yet. I intend that it will be. However, the problem is that the Jaeger UI does not accept ndjson files and also the data the UI might expect might be in a slightly different format.

I was looking into writing a "replay" script that runs through each line in the ndjson file and exports the data to the Jaeger server through the `otlp` exporter. That way we can export the data into the jaeger server to later visualization.

I was also interested at some point to explore https://zipkin.io/ for telemetry data - The openchallenges stack currently uses it. Zipkin may have better tools to analyze the data.

Reply:
Interesting! Would we have to replace the `opentelemetry` code in here?

Reply:
> Would we have to replace the opentelemetry code in here?

No. Since the data is written into the OTEL format there would be no code changes.